discover/portal/articles/130-forms/01-creating-forms.markdown

```diff
@@ -0,0 +1,147 @@
+
+    Make sure you turn on the *Required Field* selector. You want to make sure
+      that, at a minimum, you capture whether the guests had a favorable stay or
+        not.
+
+    Leave the manual option checked for creating the list of selections. If
+      you're interested in populating the field from a data provider, read the
+        article on data providers.
+
+    Add these options: *It was out of this world!*, *I had a good time.*, *I'd
+      rather go to the beach*, and *I'll never come back.* You'll notice that
+        typing in one of the fields automatically adds another blank selection line.
+        Just leave the last one blank when you're done.
+
+6.  Close the sidebar when you're done configuring the field. To see additional
+    options, click *Properties*. 
+
+7.  Now you need a text field, so add one using the same procedure you used for
+    the select field.
+
+    Configure the field as follows:
+
+    **Label:** *Want to tell us more?*
+
+    **Help Text:** Leave this blank again so the form has a consistent look.
+
+    **My text field has:** Choose *Multiple Lines*. Let guests prattle on about
+    their stay if they want to.
+
+    **Required Field:** Leave this unselected. Only require guests to fill out
+    the select field and leave this one as optional.
+
+8.  Close the sidebar when you're finished configuring the text field.
+
+9.  Back on the form builder page, you can see the way the fields will be laid
+    out. This view shows the first (and only in this case) page of the form.
+
+    ![Figure 4: The form builder page lets you preview your form layout, add a page to the form, or add some more fields.](../../images/forms-form-builder.png)
+
+10.  This form is finished, so click *Save Form*. It's also autosaved every
+     minute by default.
+
+11.  Click *Publish Form*. A dedicated URL to the form is generated, but nobody
+     has the URL yet.
+
+Now your form can be added to a page, and Lunar Resort guests can be emailed and
+provided with a link to the page where the form is displayed.
+
+## Accessing Forms
+
+Once the form is developed and published, there are two options for getting the
+published form to targeted users:
+
+1. Place the Form portlet on a site page. This approach allows users navigate
+   to the page in the site.
+
+2. Copy the dedicated form URL and provide it to users (for example, via email).
+   This limits access to the form to only those users who have the direct link.
+
+To display the form on a site page in The Lunar Resort site:
+
+1.  Add a page to the site (choose Full Page Application for the page type if
+    you only want the form application on the page). Call it *Guest Survey
+    Page*.
+
+    ![Figure 5: Add a page for guests to view and fill out your new form.](../../images/forms-guest-survey-page.png)
+
+2.  Add the Form portlet to the page. If you used a full page application, click
+    the Configure Page button (![Page Gear](../../images/icon-page-gear.png))
```

Comment:
@rbohl, what was the portal version you used to take these screenshots? I'm asking this because I'm reviewing this doc based on the master version of the portal and I didn't find this gear icon next to the word Configure, please take a look below:

![screen shot 2018-05-15 at 11 09 33 am](https://user-images.githubusercontent.com/6729287/40061988-855ba05c-5830-11e8-8b73-d176c62877bf.png)